test/components/trends/common/BackgroundWithTargetRange.test.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React from 'react';
+
+import { mount } from 'enzyme';
+
+import { formatClassesAsSelector } from '../../../helpers/cssmodules';
+
+import * as scales from '../../../helpers/scales';
+const {
+  trendsHeight,
+  trendsWidth,
+  trendsXScale: xScale,
+  trendsYScale: yScale,
+} = scales.trends;
+import SVGContainer from '../../../helpers/SVGContainer';
+
+import { TWENTY_FOUR_HRS } from '../../../../src/utils/datetime';
+
+import BackgroundWithTargetRange
+  from '../../../../src/components/trends/common/BackgroundWithTargetRange';
+import styles
+  from '../../../../src/components/trends/common/BackgroundWithTargetRange.css';
+
+describe('BackgroundWithTargetRange', () => {
+  let wrapper;
+  const props = {
+    bgBounds: {
+      veryHighThreshold: 300,
+      targetUpperBound: 180,
+      targetLowerBound: 80,
+      veryLowThreshold: 55,
+    },
+    margins: {
+      top: 0,
+      right: 0,
+      bottom: 0,
+      left: 0,
+    },
+    svgDimensions: {
+      width: trendsWidth,
+      height: trendsHeight,
+    },
+    xScale,
+    yScale,
+  };
+
+  before(() => {
+    wrapper = mount(
+      <SVGContainer dimensions={{ width: trendsWidth, height: trendsHeight }}>
+        <BackgroundWithTargetRange {...props} />
+      </SVGContainer>
+    );
+  });
+
+  it('should render two rects above and below target range', () => {
+    const nonTargetRects = wrapper.find(formatClassesAsSelector(styles.nonTargetBackground));
+    expect(nonTargetRects).to.have.length(2);
+    // Enzyme forEach cannot be replaced by _.forEach
+    // eslint-disable-next-line lodash/prefer-lodash-method
```

Comment:
I took the opposite conclusion from our chat, just disable `forEach` as necessary (i.e., line-by-line). We open the door to inconsistency much more with `forEach` since we might use the native method for setting up test fixtures or whatever. But I don't have super strong feelings. Do you have a strong preference for sticking it in the ESLint config @krystophv?